### DISCUSSION TITLE
Remove Indexer Integration

### DISCUSSION THREAD

**Discussion by Safihre** [sabnzbd/sabnzbd#2118] [Ideas]
I really wonder if anyone still uses the "Indexer Integration". It was once build by OZnzb, but since then it has not gotten much attention.
In the past I have taken several steps to try to get the usage up, but it seems not with much success:

- Updated documentation for indexers, how to implement it.
- Updated visibility in the interface.
- Reddit posts.
- Direct contact with indexer-developers.

I will now add a note to 3.6.0 asking for users that have it _enabled_ to report here if they still use it and which indexers support it.
Based on that we can decide for 3.7.0 to keep it or not.

**Comment by anonvoy** [comment]
I have it enabled and I occasionally used it. There were three indexers that supported it IIRC - one was nzbgeek, but they seem to have dropped this in their code rebuild after their hack. The other one was Usenet Crawler, that one is up and down like a rollercoaster, right now it's up, but the api is not working, and I don't see any of the required meta tags in their nzb files, so that one is probably history too. OZNZB, the original implementer, is still around and as far as I can see still offers the feedback function - but sabnzbd gives me "Problem accessing ratings server: api.oznzb.com", so there's probably a problem there too.

**Comment by thezoggy** [comment]
the indexer integration made a bit more sense when it first came about (when community driven feedback of matrix type sites was starting to take off, but then all the *adarr apps came out with much better complete/failed handling) but honestly probably do more harm than good with just added load to indexers. plus this feature always rubbed certain indexers wrong because it made it look like sab was playing favorites. personally I say remove it.. 3.6?

**Comment by jnagy129** [comment]
I have the api from nzbgeek integrated..the pull from cart feature. Is this what you are talking about here? If so, I heavily use this feature.

**Reply by Safihre** [reply]
No, that's RSS feeds. 
This is the feature that in the History you can rate the download for audio and video quality or send comments about the job. It will show special icons for that. Also in the queue you'll already see it.

**Comment by jonlbauer** [comment]
I have it enabled, but to be honest - no idea if I'm actually using it or if it's providing any real benefit for me...

- Jon

**Reply by Safihre** [reply]
Which indexers do you use it with? Or which indexers have you seen it active with (based on the 🔈 and 🎥 icons in Queue/History)?

**Reply by jonlbauer** [reply]
For usenet, I use NZBfinder andNZBGeek. Sorry - I don't ever recall seeing those icons in the queue/history... I might have missed it though.

I don't have "Remove Completed" checked in Sonarr, but I don't have any history saved to see if I can determine if it's ever been used.

- Jon

**Comment by Taomyn** [comment]
I think DOGnzb uses it, at least it's the API key I have entered in SABnzbd and is on my account's profile with them, but it was a long time ago that I would have entered it.

**Reply by Safihre** [reply]
Have you ever seen the icons in the queue or the history?  🔈 and 🎥

**Reply by Taomyn** [reply]
Like the person above, I don't think I've ever seen those and my current history. all completed, do not show them, just a tick at the left side.

![image](https://user-images.githubusercontent.com/4365929/159877535-58b41fa7-0bd8-4c67-ba0a-93add830d6ae.png)

**Comment by GollyJer** [comment]
I have it enabled with nzbgeek but like the others above I don't see any the 🔈 and 🎥 icons in Queue/History. I thought the filtering rules were working. (I use "pause if more thumbs down than up") but it seems it doesn't actually work.

**Comment by Safihre** [comment]
I think we can just remove it.. 
Unless anyone can find an indexer still supporting it.

**Reply by Safihre** [reply]
I removed it.
